docs/comb-ssi/universal-gates.md

```diff
@@ -149,5 +149,29 @@ Notice that there are input elements that are present in the negative form, name
 
 <iframe width="600px" height="300px" src="https://circuitverse.org/simulator/embed/97663" id="projectPreview" scrolling="no" webkitAllowFullScreen mozAllowFullScreen allowFullScreen></iframe>
 
-
-
+{:.quiz}
+1. How many NOR gates are used to implement OR gate ?
+   1. 2
+   *  1
+   *  3
+   *  0 
+2. How many NAND gates are used to implement OR gate ? 
+   1. 3
+   *  2
+   *  0
+   *  1 
+3. The NAND gate is AND gate followed by 
+    1. NOT Gate
+    *  OR Gate
+    *  AND Gate
+    *  XOR Gate 
+4. Digital circuit can be made by the repeated use of 
```

Comment:
This question also has plagiarism: https://www.examveda.com/digital-circuit-can-be-made-by-the-repeated-use-of-47825/
https://www.toppr.com/ask/question/digital-circuit-can-be-made-by-repetitive-use-of/